RallyHereDebugTool/Source/Private/RHDTW_Analytics.cpp

```diff
@@ -408,7 +408,7 @@ void FRHDTW_Analytics::DoTimelinePlot(FRH_WebRequests* WebRequestsTracker, const
 	TimelineCountByTimeAndKey.SetNum(SecondsInOneMinute);
 	TArray<FName> ItemsByFName;
 	TDoubleLinkedListIterator<TDoubleLinkedList<TSharedPtr<FRH_WebRequest>>::TDoubleLinkedListNode, TSharedPtr<FRH_WebRequest>>  requestsIterator(TrackedRequests.GetTail());
```

Comment:
This type definition is the reason `auto` was made

Reply:
.GetTail() doesnt return an iterator, it returns a node that is used to initialize the iterator.  So auto wouldnt work here (I tried it =D)